### ISSUE TITLE
Review and remove companies that are not currently remote

### ISSUE THREAD

**Issue by alessandromr** [italiaremote/awesome-italia-remote#262]
I noticed a trend of companies going back to the office full-time. Should we review the companies on the list and remove the ones, not in line with the repo philosophy ?

Let me know with comments or 👍 👎

**Comment by guerinoni** [comment]
For me it's worth remove all no-remote company because the repository is called `remote` not hybrid :)

**Comment by alessandromr** [comment]
@guerinoni thanks for the feedback, that is indeed what I'm thinking as well.
I'm trying to find time to do this and at the same time review the current companies to make sure they are still remote.

Also, I'm thinking of explicitly require to have the fact that the company is remote in the about us / contatti / careers.
Without the explicit phrase when no job opening are active you can't make sure of the remoteness of the company

**Comment by guerinoni** [comment]
> @guerinoni thanks for the feedback, that is indeed what I'm thinking as well. I'm trying to find time to do this and at the same time review the current companies to make sure they are still remote.
> 
> Also, I'm thinking of explicitly require to have the fact that the company is remote in the about us / contatti / careers. Without the explicit phrase when no job opening are active you can't make sure of the remoteness of the company

Remove if in company page isn't mentioned or in job offer description... Because in this page at least half of company aren't remote

**Comment by ilmanzo** [comment]
I agree with @guerinoni to remove hybrid, because it's offered by many places, this repository is about remote jobs, and also with hybrid we'd need at least to know the office location. 

As an alternative to removal, I'd suggest to introduce a new remote policy like `not anymore` so we know that the company once hired fully remote, nowadays things changed but maybe in the future will be possible.

**Comment by ldematte** [comment]
I came to this issue as I was thinking to open a new one :)
CONTRIB does not specify how to remove a company that no longer offers remote positions. I wanted to remove Amazon (I had to _leave_ because they were forcing me back to a 300+ km distance office), but I don't know how I can.  
The "not anymore" policy does sound OK-ish I think, not much because in the future it might be possible again, but more for tracking and reference.

**Comment by edoardocostantinidev** [comment]
@ldematte we can improve the contribution for sure. IMHO we don't need to track what company offered remote work, there are commits for that. I suggest just deleting it altogether in a dedicated PR.

**Comment by edoardocostantinidev** [comment]
I too am not a fan of the `Hybrid` remote policy to be honest.

**Comment by edoardocostantinidev** [comment]
What if we divide the list into sections based on remote policy with `Full` on top? What do you guys think?

**Comment by alessandromr** [comment]
Partially done in https://github.com/italiaremote/awesome-italia-remote/pull/366
